w01-controlling-the-dom/d4-dawn-intro-jquery/exercises.md

```diff
@@ -0,0 +1,35 @@
+# jQuery and DOM Exercises
+
+### Docs & Resources
+
+* [Document Object Model docs (Mozilla)](https://developer.mozilla.org/en-US/docs/Web/API/document)
+* [Document Object Model docs (W3Schools)](http://www.w3schools.com/jsref/dom_obj_document.asp)
+* [List of DOM Events](https://developer.mozilla.org/en-US/docs/Web/Events)
+
+### Exercise 0: Traversing the DOM
+
+1. Go to https://generalassemb.ly and open your Dev Tools. Select the `html` element and look at its children. What type of data structure of this collection of elements?
+1. Select the `body` element and save it as a variable. Now you can use this variable to play with jQuery traversal methods.
+1. Find the children of the header with a class of `welcome-text`.
+1. Find the parent of first `<p>` tag.
+
+### Exercise 1: Manipulating the DOM
+1. Put some DOM elements in your `index.html`. Make sure they have classes and/or ids.
+1. Manipulate one of these elements using `.append()`.
+1. Manipulate a different element using `.css()`.
+1. Use `.hide()` to make elements disappear. Inspect this element in dev tools. What is this method doing to the css?
+1. Go to the [jQuery docs](http://api.jquery.com/) and pick 3 functions we haven't covered yet. Implement them in your app!
+
+### Exercise 2: Make a ToDo List
+Use the same project that you used for Exercise 1. Remember to use Bootstrap classes so that it looks nice!
+1. Create 2 text input elements: one for the todo title, the other for description. *Hint: create a todo object in your js that has title and description attribute.*
+1. Add a submit button. *Hint: giving your element an id or class will make it easier to select with jQuery*
+1. Add an event listener to the submit button with a callback function that appends the input text to the page when the button is clicked. *Hint:* use `.val()` to get text from the input.
```

Comment:
they don't currently get `callback`; are we going to have a little more explanation in class about this?